src/components/organisms/AssetContent/MetaFull.tsx

```diff
@@ -23,10 +23,6 @@ export default function MetaFull(): ReactElement {
       /> */}
 
       {/* TODO: remove those 2 date items here when EditHistory component is ready */}
```

Comment:
so looks like we can remove this code comment then

Reply:
done! think it's good to go now

Reply:
hey, you found another comment too. I meant this comment here which we can also remove